test/runtests.jl

```diff
@@ -42,7 +42,7 @@ end
     pacman = GymEnv("MsPacman-v4")
     pacmannf = GymEnv("MsPacmanNoFrameskip-v4")
     cartpole = GymEnv("CartPole-v0")
-    bj = GymEnv("Blackjack-v0")
+    bj = GymEnv("Blackjack-v0", stateT=PyAny)
```

Comment:
For BlackJack, I don't know - I'll check. 

As a default for envs? PyArray will be much quicker I believe, because getindex(::PyArray, ...) is just a pointer lookup. It's also much more appropriate for, e.g., atari games, which return images (frames from the screen).

Reply:
PyVector is faster than PyAny for just stepping the environment:
```
Trial 1 (PyVector one is the second one):
env.pyenv: PyObject <gym.envs.toy_text.blackjack.BlackjackEnv object at 0x1266ec470> num_eps: 30000 t: 0.9989846660000005 steps: 41180
microsecs/step (lower is better): 24.25897683341429
------------------------------
env.pyenv: PyObject <gym.envs.toy_text.blackjack.BlackjackEnv object at 0x1266f95c0> num_eps: 30000 t: 0.7416911469999994 steps: 41335
microsecs/step (lower is better): 17.943417128341586

Trial 2:
env.pyenv: PyObject <gym.envs.toy_text.blackjack.BlackjackEnv object at 0x129ae4a20> num_eps: 30000 t: 0.8304164830000038 steps: 41227
microsecs/step (lower is better): 20.142539670604307
------------------------------
env.pyenv: PyObject <gym.envs.toy_text.blackjack.BlackjackEnv object at 0x129ae4b70> num_eps: 30000 t: 0.737025869000004 steps: 41307
microsecs/step (lower is better): 17.842638511632508
```
dunno about doing stuff with the PyVector, as compared to the `Tuple{Int64,Int64,Bool}`, but don't really care either :trollface:

Reply:
:ok_hand:

Reply:
btw blackjack is faster than python in these benchmarks because the step speed is so quick that julia's loop speed becomes a factor

Reply:
At least I figure that's what's happening.